static/css/global-OLD.css

```diff
@@ -0,0 +1,85 @@
+body {
```

Comment:
If I had FF flags in mind before I made this PR I would've structured the changes in such a way that we don't need 2 copies of this file. Working backwards it's easier & less prone to issues if we just have 2 copies.